test/units/modules/remote_management/lxca/test_lxca_nodes.py

```diff
@@ -0,0 +1,110 @@
+import json
+
+from ansible.compat.tests import unittest
+from ansible.compat.tests.mock import patch, Mock
+from ansible.module_utils import basic
+from ansible.module_utils._text import to_bytes
+from ansible.modules.remote_management.lxca import lxca_nodes
+
+import mock
+
+
+def set_module_args(args):
+    """prepare arguments so that they will be picked up during module creation"""
+    args = json.dumps({'ANSIBLE_MODULE_ARGS': args})
+    basic._ANSIBLE_ARGS = to_bytes(args)
+
+
+class AnsibleExitJson(Exception):
+    """Exception class to be raised by module.exit_json and caught by the test case"""
+    pass
+
+
+class AnsibleFailJson(Exception):
+    """Exception class to be raised by module.fail_json and caught by the test case"""
+    pass
+
+
+def exit_json(*args, **kwargs):
+    """function to patch over exit_json; package return data into an exception"""
+    if 'changed' not in kwargs:
+        kwargs['changed'] = False
+    raise AnsibleExitJson(kwargs)
+
+
+def fail_json(*args, **kwargs):
+    """function to patch over fail_json; package return data into an exception"""
+    kwargs['failed'] = True
+    raise AnsibleFailJson(kwargs)
+
+
+def fake_conn(*arg, **kwargs):
+    return "Fake connection"
+
+
+class TestMyModule(unittest.TestCase):
```

Comment:
New tests must use `pytest` instead of `unittest` ([see](https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_general.html?highlight=pytest#unit-tests))

Just in case, here are some tests using `pytest`:
- https://github.com/ansible/ansible/blob/devel/test/units/modules/net_tools/test_nmcli.py
- https://github.com/ansible/ansible/pull/46590/files

Reply:
I used unittest as mock sample page https://docs.ansible.com/ansible/devel/dev_guide/testing_units_modules.html#use-of-mocks still uses unittest.TestCase.  Using pytest command we can run unit tests which are using unittest.TestCase

Reply:
Updated tests with pytest  fixtures and parameterized tests